bin/riak-bucket-exporter.js

```diff
@@ -111,6 +119,11 @@ function processKey(key, cb) {
     }
     fs.appendFileSync(program.file, JSON.stringify(out,null,'\t'));
     first=false;
+    
+    if (deleteKeys) {
+        db.remove(bucket, key);
```

Comment:
thanks for this pull request. Please don't do a fire and forget here, as this may create a large number of parallel open http delete requests we cannot control. could you please change the pull request here.